apps/web/components/eventtype/EventAvailabilityTab.tsx

```diff
@@ -118,10 +118,12 @@ const EventTypeScheduleDetails = memo(
                     {filterDays(index).map((dayRange, i) => (
                       <div key={i} className="text-default flex items-center leading-4">
                         <span className="w-16 sm:w-28 sm:text-left">
-                          {format(dayRange.startTime, timeFormat === 12)}
+                          {format(dayRange.startTime, timeFormat === 12).replace(/24(?=:)/g, "00")}
```

Comment:
let's make this directly in the format function (https://github.com/calcom/cal.com/pull/10661/files#diff-99c36cb0362623749ba84d29249cbb05823ac6fb0d8dad23efa322e063121accR67)